src/effects/effectbuttonparameterslot.cpp

```diff
@@ -23,7 +23,7 @@ EffectButtonParameterSlot::EffectButtonParameterSlot(const QString& group,
             ConfigKey(m_group, itemPrefix + QString("_type")));
 
     connect(m_pControlValue, SIGNAL(valueChanged(double)),
```

Comment:
move to EffectParameterSlotBase

Reply:
We cannot shift `m_pControlValue` to EffectParameterSlotBase.